cpp/src/parquet/encoding.cc

```diff
@@ -2537,6 +2537,102 @@ class DeltaBitPackDecoder : public DecoderImpl, virtual public TypedDecoder<DTyp
 // ----------------------------------------------------------------------
 // DELTA_LENGTH_BYTE_ARRAY
 
+// ----------------------------------------------------------------------
+// DeltaLengthByteArrayEncoder
+
+template <typename DType>
```

Comment:
Seems our standard says DeltaLengthByteArray only supports BYTE_ARRAY, can we use a fixed type or limit the `DType` here?

Reply:
I removed the template logic, so only `ByteArrayType` is left.